### ISSUE TITLE
I wonder how can u figure out the architecture of PixColor.(especially about Refinement network)

### ISSUE THREAD

**Issue by demul** [sjooyoo/pix-color_pixel-cnn#2]
Hi i'm trying to implement PixColor on tensorflow. but I have a problem.

The paper only describe the refinement architecture like that "It is similar in architecture to the network used in [15] but with more layers in decoding part. and use bilinear upsample instead of transposed-conv upsample".

I couldn't know how many layers stacked additionally. and I couldn't know about training hyperparameters(Initial learning rate, Weight decaying, What solver they use, etc....)

then, how can u deal with that problems?
